indico/modules/categories/models/categories.py

```diff
@@ -335,6 +335,65 @@ def parent_chain_query(self):
         """
         return self._get_chain_query(Category.id == self.parent_id)
 
+    def nth_parent(self, n_categs, fail_on_overflow=True):
+        """Return the nth parent of the category.
+
+        :param n_categs: the number of categories to go up
+        :param fail_on_overflow: whether to fail if we try to go above the root category
+                                 (or provide a negative ``n_categs``)
+        :returns: ``Category`` object or None (only if ``fail_on_overflow`` is not set)
```

Comment:
`:return:` is more common than `:returns:` and while we don't use sphinx right now, the class name is fine in single backticks (that way it'd be linked to the object's documentation)